src/neuroconv/basedatainterface.py

```diff
@@ -20,7 +20,10 @@
 class BaseDataInterface(ABC):
     """Abstract class defining the structure of all DataInterfaces."""
 
+    display_name: Union[str, None] = None
     keywords: List[str] = []
+    common_suffixes: List[str] = []
+    help: Union[str, None] = None
```

Comment:
Any reason you aren't using Optional here?

Reply:
@h-mayorquin and I have gone back and forth on this philosophy - knowing to use `Optional` because it is equivalent is an 'implicit' behavior (PEP8 says explicit > implicit) and therefore not always obvious at first glance or to newcomers, or to even seasoned devs who forget in a moment of debugging

So we find it a bit easier to read to see at a glance the `Union` of all allowed types for the value

Also in (3.11 I think?) this syntax becomes `str | None` so it's even cleaner than `Optional` wrapping at that point too